tagging.py

```
@@ -1,16 +1,503 @@
+        if token_type in (ItemType.Honorific, ItemType.InfoSpecifier):
+            lex.accept(".")
+        lex.emit(token_type)
+    elif cal(word):
+            lex.emit(ItemType.Calendar)
+    else:
+            lex.emit(ItemType.Text)
+    return run_lexer
+    
+
+            
+def lex_number(lex: Lexer) -> LexerFunc | None:
```

Flake8 (7.2.0)

[error] too many blank lines (3)

(E303)